src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowJob.java

```diff
@@ -656,7 +657,9 @@ public static void alias() {
         WorkflowRun.alias();
     }
 
-    @Extension(ordinal=1) public static final class DescriptorImpl extends TopLevelItemDescriptor {
+    @Extension(ordinal=1)
+    @Symbol("pipeline")
```

Comment:
FreeStyle has both `freeStyle` and `freeStyleJob`   not clear to me what the convention here is, should we be including `"Job"` or not?

Reply:
Neither do I know, so I've opted for pipeline since it's popularly known that way. I can add both if desired